Fixing some typos in PR template (#170)
NethermindEth/Forta-Agents: .github/pull_request_template.md

- [ ] The agent README sufficiently describes what the agent does
- [ ] All the alerts that the agent emits are listed in the README
- [ ] The txn/block examples listed in the README generate the expected findings
- [ ] Code is easy to follow and/or properly commented
- [ ] Agent contains tests

> - [ ] All tests pass when running `npm tests`

Reviewer: ```suggestion
- [ ] All tests pass when running `npm test`
```

- [ ] Tests are not making real network calls (i.e. network interaction is properly mocked)
- [ ] Tests cover all the paths (i.e. negative & positive test cases)
- [ ] When the tests complete, no warning/errors are shown
- [ ] `npm start` works without errors when executed using a jsonRpcUrl for each of the networks specified in the README
- [ ] The code does what the README describes
- [ ] Network calls are properly being cached (if duplicated calls can occur)
- [ ] A performance review has been done (e.g. `Promise.all` is used whenever possible)
- [ ] The findings fields are filled out correctly

> - [ ] Findings `protocol` is specified (if the agent is for an specific one)

Reviewer 2: ```suggestion
- [ ] Findings `protocol` is specified (if the agent is for a specific one)
```

- [ ] `metadata` of the findings contains all the information needed.
- [ ] The SDK methods are being used appropriately (e.g. `filterLog`, `filterFunction`, `getEthersProvider`)
- [ ] `ABI`s match the contracts used / `args` used from `LogDescriptions` exists in the `ABI`s

> - [ ] the agent follows Forta recommended [best practices](https://docs.forta.network/en/latest/best-practices/)

Reviewer 2: ```suggestion
- [ ] The agent follows Forta recommended [best practices](https://docs.forta.network/en/latest/best-practices/)
```

> - [ ] Agent can recover from failed error call when they can fail (e.g. error are being catched on calls that can fail)

Reviewer 2: ```suggestion
- [ ] Agent can recover from failed error calls when they can fail (e.g. errors are being caught on calls that can fail)
```
